frontend/package.json

```diff
@@ -6,26 +6,23 @@
     "@ant-design/charts": "1.3.6",
     "@ant-design/colors": "6.0.0",
     "@ant-design/icons": "4.7.0",
-    "@craco/craco": "6.4.4",
     "@react-spring/web": "9.4.5",
     "@reduxjs/toolkit": "1.8.3",
     "@tippyjs/react": "4.2.6",
+    "@vitejs/plugin-react-refresh": "^1.3.6",
     "antd": "4.21.5",
     "axios": "0.27.2",
-    "craco-less": "1.20.0",
     "framer-motion": "6.4.2",
     "moment": "2.29.3",
     "react": "17.0.2",
```

Comment:
We should attempt to update this as well as the other dependencies we've blocked

Reply:
I realised this when removing cracro but the issue with updating to react 18 is that `react-beautiful-dnd` (our drag and drop library) only officially supports upto react 17. And it seems based on their github that its not currently being maintained by atlassian at the moment 😢. We can try using legacy peer deps but that seems such a workaround and my some unknown bugs.

Reply:
Smh @SpanishPear plz fix

Reply:
[This](https://github.com/atlassian/react-beautiful-dnd/issues/2399) issue suggest that it does work in react 18, just not in strict mode StrictMode

Reply:
Hello! I just released a fork of `react-beautiful-dnd` which should fully support react 18 and react strict mode.

Just replace `react-beautiful-dnd` with `@hello-pangea/dnd` and you should be good to go 👍